.travis.yml

```diff
@@ -33,7 +33,7 @@ script:
 - cargo build --verbose --features common
 - cargo test --verbose --features common
```

Comment:
Should we maybe add another test for building with rudy?

Reply:
There is already commands to build and test with rudy.